README.md

```diff
@@ -415,6 +446,16 @@ This method makes a call to the
 [/{db}/_design/{ddoc}/_view/{view}](http://docs.couchdb.org/en/1.6.1/api/ddoc/views.html#get--db-_design-ddoc-_view-view)
 endpoint, and the builder supports all the relevant options.
 
+Similarly, to query the total age of Persons in the document using the
+`totalAgeView` builder we can do:
+
+```Scala
+totalAgeView.queryWithReduce[Int]
```

Comment:
I just realized, do we really need to specify the reduce function return type here? Since we already passed the `Int` type in `db.query.view[String, Int]`, can that type be used instead?